### ISSUE TITLE
Is this project still alive?

### ISSUE THREAD

**Issue by schadom** [UFAL-DSG/cloud-asr#13]
cloudasr.com seems to be dead. Thanks

**Comment by choko** [comment]
Hi,

I am sorry, the service www.cloudasr.com is down as we don't have any servers to host it. The code and models should work and we might release new models next year. Are you interested in the service or do you want to run it on your servers?

Ondrej

**Comment by oplatek** [comment]
@choko 
@schadom 
We may be able to host it with a demo on our company website.

Can you tell me what is your use-case for it?

**Comment by schadom** [comment]
Hi @oplatek,
currently no concrete use-case besides playing around with different ASR technologies.

**Comment by dpny518** [comment]
@oplatek i read your thesis, and this was able to get 1000 parallel request? Do you know the most recent updated version, I want to try and put it up and see if I can use it for real time production use

**Comment by oplatek** [comment]
@bamba518 it is actually @choko thesis, so let him respond.

Note: I used the code for production before.

**Comment by ondrejklejch** [comment]
@bamba518 In the thesis we used 1000 fake ASR engines, because we didn't have big enough cluster to run 1000 real ASR engines. In theory, the only limit for the number of parallel requests is a size of your cluster, because one worker can handle only one request.

**Comment by dpny518** [comment]
@choko does the size of cluster mean that each core can only handle 1 worker? so 1000 parallel request requires 1000 cores?

**Comment by ondrejklejch** [comment]
Yes, I think this is a good rough estimate - it all depends on your ASR model. Do you have a good ASR model? Is it for closed or open domain?

**Comment by dpny518** [comment]
I was just going to us this model
http://goofy.zamia.org/zamia-speech/asr-models/kaldi-generic-en-tdnn_f-r20180901.tar.xz

My goal is to handle thousands of parallel connections using http streaming, do you have any recommendations ?

**Comment by ondrejklejch** [comment]
I am not familiar with this model, but TDNN-F models should be a good choice. Make sure to rebuild the docker images, because nnet3 models are not supported in the docker images available from the docker hub.
